CHANGES.rst

```diff
@@ -9,6 +9,7 @@ Unreleased
   ``SIMPLE_HISTORY_ENFORCE_HISTORY_MODEL_PERMISSIONS`` is set to ``True``
   in ``settings`` (gh-1017).
 - Fixed ``SimpleHistoryAdmin`` not properly integrating with custom user models (gh-1177)
+- Fixes self-m2m fields not using the correct through field name (gh-1218)
```

Comment:
Something like this would probably be easier to understand for users without knowledge of this project's internal code: (do adjust if you want 🙂)
```suggestion
- Fixed ``FieldError`` when creating historical records for many-to-many fields with
  ``to="self"`` (gh-1218)
```